labgrid/resource/udev.py

```diff
@@ -40,7 +40,7 @@ def poll(self):
 class USBResource(ManagedResource):
     manager_cls = UdevManager
 
-    match = attr.ib(validator=attr.validators.instance_of(dict), hash=False)
+    match = attr.ib(default={}, validator=attr.validators.instance_of(dict), hash=False)
```

Comment:
I'm not sure about this, you could also have an explicit empty match argument if you want to match everything. It seems a little bit dangerous to make match everything implicit.

Reply:
just wanted to put it up for discussion. Im ok with putting the empty match in my yaml file but it looked to me that the USB resources already have enough internal checks which reduces the need of a mandatory match.
if you prefere to keep i will revert that change.